report.qmd

```diff
@@ -32,9 +32,41 @@ editor: visual
 
 ```
 
-## Introduction
-
-Urban areas across Europe are increasingly challenged by the impacts of climate change, biodiversity loss, and the demand for sustainable development [@nijkamp2013new]. According to [@article], in cities the impact of land use on fresh-water ecosystems is especially severe. As climate change is expected to increase the intensity and frequency of extreme weather events, solutions to combat these effects are urgently needed [@ourloglou2020assessing].
+**Table of contents:**
+
+1.  Introduction;
+2.  Methodology;
+    a.  Definition of Spatial Units;
+    b.  Resolution & Data Acquisition Method
+    c.  Scale-Specific Indicators for Stream Assessment
+    d.  Impact of Scale on Detectable Effects
+    e.  Normalization;
+3.  Spatial Multi-Criteria Decision Analysis (S-MCDA)
+    A.  Biodiversity;
+        a.  Which Areas Have High Biodiversity Value?
+        b.  Which Areas Have a High Level of Human Activity?
+        c.  Which Areas Provide Opportunity for Improved Biodiversity?
+    B.  Climate Adaptation;
+        a.  Which Areas Have Highest Flood Damage Risk?
+        b.  Which Areas Have The Most Impervious Surfaces?
+        c.  Which Areas Have Healthy Vegetation?
+    C.  Quality of Life;
+        a.  Which Areas Are Deprived of Public Facilities?
+        b.  Which Areas Lack Infrastructure to Access?
+        c.  Which Areas Can Be Physically Uncomfortable?
+    D.  Conclusion of S-MCDA;
+4.  K-Means Clustering Analysis;
+    a.  Research question;
+    b.  Methodology;
+    c.  Variables;
+    d.  Results;
+5.  Discussion;
+6.  Conclusion;
+7.  Appendix.
```

Comment:
Alternatively, let quarto generate your TOC, as shown here: https://quarto.org/docs/output-formats/html-basics.html#table-of-contents